src/test/java/com/exasol/udfstartuptimeimprover/UdfStartUpTimeImproverIT.java

```diff
@@ -0,0 +1,144 @@
+package com.exasol.udfstartuptimeimprover;
+
+import static com.exasol.udfstartuptimeimprover.UdfStartUpTimeImprover.*;
+import static org.hamcrest.MatcherAssert.assertThat;
+import static org.hamcrest.Matchers.equalTo;
+import static org.junit.jupiter.api.Assertions.assertAll;
+import static org.junit.jupiter.api.Assertions.assertDoesNotThrow;
+
+import java.io.IOException;
+import java.io.InputStream;
+import java.nio.charset.StandardCharsets;
+import java.nio.file.Path;
+import java.sql.*;
+import java.util.Objects;
+import java.util.concurrent.TimeoutException;
+
+import org.jetbrains.annotations.NotNull;
+import org.junit.jupiter.api.*;
+import org.junit.jupiter.api.io.TempDir;
+import org.testcontainers.junit.jupiter.Container;
+import org.testcontainers.junit.jupiter.Testcontainers;
+
+import com.exasol.bucketfs.Bucket;
+import com.exasol.bucketfs.BucketAccessException;
+import com.exasol.containers.ExasolContainer;
+import com.exasol.dbbuilder.dialects.exasol.*;
+import com.exasol.dbbuilder.dialects.exasol.udf.UdfScript;
+import com.exasol.mavenprojectversiongetter.MavenProjectVersionGetter;
+import com.exasol.udfdebugging.UdfTestSetup;
+
+@Testcontainers
+class UdfStartUpTimeImproverIT {
+    @Container
+    private static final ExasolContainer<? extends ExasolContainer<?>> EXASOL = new ExasolContainer<>().withReuse(true);
+    private static final String CURRENT_VERSION = MavenProjectVersionGetter.getCurrentProjectVersion();
+    private static final String BUCKET_FS_CONNECTION = "MY_BUCKET_FS_CONNECTION";
+    private static Connection connection;
+    private static Statement statement;
+    private static Bucket bucket;
+    private static ExasolSchema schema;
+
+    @BeforeAll
+    static void beforeAll()
+            throws SQLException, BucketAccessException, IOException, TimeoutException, InterruptedException {
+        EXASOL.getDefaultBucket().uploadFile(Path.of("udf-for-testing/target/udf-for-testing.jar"),
+                "udf-for-testing.jar");
+        EXASOL.getDefaultBucket().uploadFile(
+                Path.of("target/java-udf-startup-time-improver" + "-" + CURRENT_VERSION + ".jar"), "improver.jar");
```

Comment:
```suggestion
                Path.of("target/java-udf-startup-time-improver-" + CURRENT_VERSION + ".jar"), "improver.jar");
```
or is this by intention?

Reply:
yes, it's a hack because otherwise the artifact reference checker replaces it with a hard-coded version.